addons/pos_pinelabs/static/src/app/utils/payment_pinelabs.js

```diff
@@ -0,0 +1,204 @@
+import { _t } from "@web/core/l10n/translation";
+import { PaymentInterface } from "@point_of_sale/app/payment/payment_interface";
+import { AlertDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { serializeDateTime } from "@web/core/l10n/dates";
+import { registry } from "@web/core/registry";
+
+const CANCEL_REQUEST_TIME_LIMIT = 125000;
+const { DateTime } = luxon;
+
+export class PaymentPinelabs extends PaymentInterface {
+    setup() {
+        super.setup(...arguments);
+        this.paymentInitialTime = null;
+        this.paymentLineResolvers = {};
+        this.offlineErrorHandler = registry.category("error_handlers").get("offlineErrorHandler");
```

Comment:
export `offlineErrorHandler` from the `error_handlers` module and import from here. It's not a hook, there is no point on importing it this way.